server/lease/lessor.go

```diff
@@ -324,11 +329,15 @@ func (le *lessor) Grant(id LeaseID, ttl int64) (*Lease, error) {
 }
 
 func (le *lessor) Revoke(id LeaseID) error {
+	defer func(start time.Time) {
+		leaseRevokeDurationSeconds.Observe(time.Since(start).Seconds())
```

Comment:
How this differs than grpc latency metrics for "leaseRevoke" request?

Reply:
Ack, don't need this in favor of `grpc_server_handling_seconds_bucket{grpc_method="LeaseRevoke"...}`. Will update PR.